### DISCUSSION TITLE
【建议】想随便聊一聊几个小的优化方向

### DISCUSSION THREAD

**Discussion by Hongxuli2006** [share121/inter-knot#1508] [其他]
![邦布(1)](https://github.com/user-attachments/assets/9805c16f-f83c-4426-be6d-347eac4f8b09)

绳网越来越好了，真的佩服大佬的能力。不过有这么几个想说的可优化的方向，第一个就是关于上面那一行数字
![屏幕截图 2024-08-11 203939](https://github.com/user-attachments/assets/b952142a-8a28-47c4-89c4-569364d8b1f4)
今天上午我记得那一排数字是随机的，不知道现在更新了之后还是不是这样子的

其二就是关于UI设计，现阶段绳网暂时是没有等级提升功能！期待后续会增加！
最后就是提醒各位绳匠，注意把不想关注的话题评论后取消关注。否则就会像我这样子

![屏幕截图 2024-08-11 203104](https://github.com/user-attachments/assets/27a6b919-f3ea-48c9-9c71-9df07b56b16a)
一时间来这么多消息我还以为我爆仓了哈哈哈哈。吓死我了

**Comment by VacuolePaoo** [comment]
是随机的，绳网等级对应的是仓库的数量，而且你任务栏的clash露出来了，请及时更新

**Reply by Hongxuli2006** [reply]
好嘞！谢谢提醒

**Comment by capacitor1** [comment]
🤔邮箱啊 我从来不看邮箱doge

> 其二就是关于UI设计，现阶段绳网暂时是没有等级提升功能！期待后续会增加！

实际上现在可以升级。。只是升级的方法和绳网没有一点关系罢了

**Reply by VacuolePaoo** [reply]
不然勒，在浏览器里使用HDD？

**Reply by Hongxuli2006** [reply]
> 🤔邮箱啊我从来不看邮箱doge
> 
> > 其二是关于UI设计，现阶段绳网暂时是没有等级提升功能！后续会增加！
> 
> 注意现在可以升级。只是升级的方法和绳网没有一点关系没关系
我是因为工作的原因必须看邮箱，而且我人因为在美国的原因也必须得看哈哈哈。（虽然说现在不在）
